resources/css/education.css

```diff
@@ -0,0 +1,182 @@
+@font-face {
+  font-family: OpenSans-SemiBold;
+  src: url(fonts/OpenSans-SemiBold.ttf)
+}
+
+a, a:visited {
+  color:black;
+  text-decoration: none;
+  padding: 20px;
+}
+
+ul {
+  list-style-type: none;
+}
+
+li {
+  display: inline-block;
+  margin-left: 17px;
+  margin-right: 17px;
+}
+
+.scrollPage {
+  overflow: auto;
+}
+
+.header {
+  font-family: OpenSans-SemiBold;
+  font-size: 22px;
+  display: table;
+  margin: auto;
+}
+
+.space {
+  grid-area: space;
+}
+
+.lets-deal {
```

Comment:
We are making CSS class names all camel case for consistency. This should changed (here and in the .blade.php file!) to be .letsDeal